src/Components/CreateImageWizard/ChippingInput.tsx

```diff
@@ -64,7 +66,7 @@ const ChippingInput = ({
   };
 
   const handleKeyDown = (e: React.KeyboardEvent, value: string) => {
-    if (e.key === ' ' || e.key === 'Enter' || e.key === ',') {
+    if (e.key === ' ' || e.key === 'Enter') {
```

Comment:
Comma should be a valid character for kernel argument, I've removed it from the keyDown for the consistency sake.